src/Model/User.php

```diff
@@ -96,6 +96,59 @@ class User
 
 	private static $owner;
 
+	/**
+	 * Sends a welcome e-mail to the specified user
+	 *
+	 * @param array $user The user
+	 * @param string $password The user's password
+	 * @param string $preamble The user's password
+	 *
+	 * @return NULL|boolean from notification() and email() inherited
+	 * @throws HTTPException\InternalServerErrorException
+	 */
+	private static function sendWelcomeEmail(array $user, string $password, string $preamble)
+	{
+		$body = DI::l10n()->t( <<<'EOT'
+			The login details are as follows:
+
+			Site Location: 1$s
+			Login Name:    %2$s
+			Password:      %3$s
+
+			You may change your password from your account "Settings" page after logging
+			in.
+
+			Please take a few moments to review the other account settings on that page.
+
+			You may also wish to add some basic information to your default profile
+			' . "\x28" . 'on the "Profiles" page' . "\x29" . ' so that other people can easily find you.
```

Comment:
concatenation don't work in nowdoc.

```suggestion
			(on the "Profiles" page) so that other people can easily find you.
```